db/migrate/20210330003039_add_index_to_tutoring_session_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddIndexToTutoringSessionUsers < ActiveRecord::Migration[6.1]
+  def change
+    add_column :tutoring_session_users, :id, :primary_key
```

Comment:
Join tables are not supposed to have ids

Reply:
An issue was occuring where the join table was being changed, and then rails yelled that it couldn't find the join table. I attempted to have it pk'd via [user_id, tutoring_session_id] however it was not working. In an ideal world, we would index off those two columns but rails sucks and it'd take too much effort compared to just adding this pk.

Reply:
Wait, did you attempt to set user_id and tutoring_session_id to primary keys, or did you index them? Because those are two different things.

Reply:
Both at some point, but I attempted in different ways to set primary_key specifically